backend/app/literature/config.py

```diff
@@ -32,7 +32,7 @@ class GlobalConfig(BaseSettings):
     PSQL_PORT: Optional[str] = None
     PSQL_DATABASE: Optional[str] = None
 
-    RESOURCE_DESCRIPTOR_URL: str = None
+    RESOURCE_DESCRIPTOR_URL: Optional[str] = None
```

Comment:
this one needs to be required. so remove the Optional and the "= None" would be correct